### ISSUE TITLE
Groups lookup plugin should take into account old taxonomy tagging

### ISSUE THREAD

**Issue by illepic** [illepic/downfall-guild#29]
Spun off from: https://github.com/illepic/downfall-guild/issues/7

Lots of old content used taxonomy terms to tag raid group content (Hamfist, Kitten Brigade, etc). With our newer, reduced taxonomy terms, we've removed all these old raid group tags.

We will now augment the existing group lookup plugin to check for any of these old group taxonomy terms and instead tag them with their appropriate Group.

The logic of the plugin looks like this:

1. Lookup any groups this content is a part of and return each of them for direct reference
1. Lookup any tags this content has that are part of the 9-10 old raid groups tags, return the nid of the group instead of the tid

Basically, if an old node was tagged as "Hamfist" as a taxonomy term, we will look up the appropriate node reference instead for the Hamfist Group node.

**Comment by illepic** [comment]
This is now working for all old content.
